docs/web-ui.md

```diff
@@ -407,6 +407,20 @@ Here is the list of SQL metrics:
 
 </table>
 
+## Structured Streaming Tab
+When running Structured Streaming jobs in micro-batch mode, a Structured Streaming tab will be
+available on the Web UI. You can check the statistics of a running query or a completed query 
+via clicking a run id on the Streaming Query overview page.
+
+<p style="text-align: center;">
+  <img src="img/webui-structured-streaming-detail.png" title="Structured Streaming Query Statistics" alt="Structured Streaming Query Statistics">
+</p>
+
+The statistics page displays some useful metrics (e.g., how many records received per second and 
+the time used for different stages of a micro-batch execution) for insight into the status of your
```

Comment:
How about adding a section for the metrics in operation duration histogram?

Reply:
+1, we can also give some information for each metrics, like:
- Input Rate: The aggregate (across all sources) rate of data arriving.
- Process Rate: The aggregate (across all sources) rate at which Spark is processing data.
- Input Rows: The aggregate (across all sources) number of records processed in a trigger.
- Batch Duration: The process duration of each batch.

Reply:
Sounds good. I'll add a list of metrics.